packages/beacon-node/src/api/impl/validator/index.ts

```diff
@@ -614,6 +572,10 @@ export function getValidatorApi(
       );
     }
 
+    const graffitiBuffer = toGraffitiBuffer(
+      graffiti ?? getDefaultGraffiti(getLodestarClientVersion(opts), chain.executionEngine.clientVersion, opts)
+    );
+
```

Comment:
we can avoid another graffiti conversion here

https://github.com/ChainSafe/lodestar/blob/3333f322d9cde1b616dec8886b619bd239ff2b02/packages/beacon-node/src/api/impl/validator/index.ts#L595-L597

Reply:
Done.